python/whylogs/viz/__init__.py

```diff
@@ -1,8 +1,8 @@
 import IPython  # type: ignore  # noqa  # import ipython to make
 
-from whylogs.viz.jupyter_notebook_viz import NotebookProfileVisualizer
```

Comment:
Will this break integrations that reference `whylogs.viz.jupyter_notebook_viz` ? NotebookProfileVisualizer should be the only thing integrations reference and should use whylogs.viz so maybe it is ok, but we might consider putting a deprecation warning there.

Reply:
discussed, and usage should be using whylogs.viz which remains the same, risk is acceptable.